package.json

```diff
@@ -28,6 +28,7 @@
     "apollo-boost": "^0.4.9",
     "date-fns": "^2.14.0",
     "graphql": "^14.6.0",
+    "moment": "^2.27.0",
```

Comment:
is there a reason we've installed moment here? I don't think we're using it anywhere

Reply:
it's used to format the date in the patient table